src/helpers/map.js

```diff
@@ -137,20 +137,19 @@ export function getTraversableCells(cpt, map) {
   }
   for (x = 0; x < xl; x++) {
     for (let y = 0; y < yl; y++) {
-      // TODO: Set radius to be correct value for each cell object. Currently using 29 because it is > 20 * sqrt(2),
-      // the furthest distance from the center of a tile to its corner, in pixels. This guarantees that if the tile
-      // has anything non-tranversable in it (wall, ball, spike, corner wall), the entire tile is marked as non-traversable
+      // TODO: Set radius to be correct value for each cell object. Currently using 29 because it
+      // is > 20 * sqrt(2), the furthest distance from the center of a tile to its corner, in
+      // pixels. This guarantees that if the tile has anything non-tranversable in it (wall, ball,
+      // spike, corner wall), the entire tile is marked as non-traversable
       const objRadius = 29;
       fillGridWithSubgrid(
         emptyCells,
         traversableCellsInTile(
           isTraversable(map[x][y]),
           cpt,
-          objRadius
-        ),
+          objRadius),
```

Comment:
@billmwong if you want thigs to be seperaionated like this, the brackets have to be at the end, or eslint complains b/c it wants a trailing comma...